app/views/ForwardLivechatView.tsx

```diff
@@ -148,6 +160,11 @@ const ForwardLivechatView = ({ forwardRoom, navigation, route, theme }: IForward
 			value: room?.departmentId,
 			data: departments,
 			onChangeValue: setDepartment,
+			onChangeText: getDepartments,
```

Comment:
We did have search on `PickerView`, but this is the first time we're using it.
So we should fix this.
Search header shouldn't be sticky to header (rendered on top of the list while scrolling).
<img src="https://user-images.githubusercontent.com/804994/150198237-2ca65fa1-5449-4bd8-92ef-9c64ba991984.png" width="350" />

It should be part of the scroll and render only on top.